### ISSUE TITLE
Add a location for 3rd party directories

### ISSUE THREAD

**Issue by ar13pit** [tue-robotics/tue-env#410]
Currently 3rd party software like openpose, torch, openface etc. are installed in the home directory. It will be better to group all software installed by tue-env.

**Comment by Rayman** [comment]
Ideally you shouldn't install these things from source. A debian package is prefered

**Comment by ar13pit** [comment]
True. But at the moment debian packages of these libraries don't exist so we are installing them from source.

I have written a dockerfile for their installation as libraries like OpenPose don't run on everyone computer, but haven't tested it thoroughly yet.
